src/lib.rs

```diff
@@ -69,7 +69,17 @@ pub trait Field:
     fn one() -> Self;
 
     /// Returns true iff this element is zero.
-    fn is_zero(&self) -> bool;
+    fn is_zero(&self) -> Choice;
```

Comment:
It occurs to me that this could also now have a provided impl:
```suggestion
    fn is_zero(&self) -> Choice {
        self.ct_eq(&Self::zero())
    }
```